reddit_decider/__init__.py

```diff
@@ -886,6 +886,39 @@ def get_all_dynamic_configs(self) -> List[Dict[str, Any]]:
 
         return parsed_configs
 
+    def get_experiment(self, experiment_name: str) -> Optional[ExperimentConfig]:
+        """Get an `ExperimentConfig` representation of an experiment or `None` if not found.
+
+        :param experiment_name: Name of the experiment to be fetched.
+
+        :return: an `ExperimentConfig` representation of an experiment if found, else `None`.
+        """
+        decider = self._get_decider()
+        if decider is None:
+            return None
+
+        experiment = decider.get_experiment(experiment_name)
+        error = experiment.err()
+        if error:
+            # sending to debug logger to avoid printing "Feature x not found." logs
+            logger.debug(f"Encountered error in decider.get_experiment(): {error}")
```

Comment:
@julieahnt printing "experiment not found" errors to debug here so as to not pollute logs

Reply:
lgtm